### PR TITLE
completed assessment

### PR DESCRIPTION
**Pull request by Mvangg**
# Submitting Week 1 Assessment

### ✅ Completed Files

Select the following as confirmation that the files in this repository are complete.

- [X] Coding Challenges
- [X] Coding Concepts
- [X] Interview Questions
- [X] Career Readiness

### ✋ Reminder

Do NOT merge your assessment!

### 📝 Feedback

If you would like specific feedback please leave details here:

### CODE REVIEW

**Review by AbdullahWins — COMMENTED** [review]
Good Job!

File: code-challenges.js [hunk 1]
```diff
@@ -10,36 +10,78 @@
...
-// Pseudo code:
+// Pseudo code: 
+// I need to combine the two arrays which are [2, 5, 2, 2, 4] and [6,, 3, 5, 3] into one array.
+// Once the two arrays are combined, I need to run a .length to count the elements/items in the final array. 
+// The output will come out as 9 since there are 9 elements/items in the whole array now.
+// The below is the process into one whole line:
+
+// console.log(padres1984WorldSeriesRuns.concat(padres1998WorldSeriesRuns).length) // --> 9
+
+// // The below is the process using a new variable to show the outcome of the combined two original variables's length.
```
Comment (chrisyhlee) [inline code-challenges.js:46]: Here is how you could rewrite this pseudo code to be more concise:
- combine the given arrays using .concat
- log the new length of the concatenated array using .length

This explains exactly what the code is doing and keeps it short and sweet!

File: coding-concepts.js [hunk 2]
```diff
@@ -7,49 +7,55 @@
...
 // --------------------5) What will this log?
 
-const dataTypes = ["number", "string", "Boolean", "undefined"]
+// const dataTypes = ["number", "string", "Boolean", "undefined"]
 // console.log(typeof dataTypes.length)
 
-// a) Your answer:
-// b) Verify and explain:
+// a) Your answer: Error?
+// b) Verify and explain: (This one took a while to understand.) typeof will let the user know what type of data the expression or value is. I assumed it would not work because dataTypes.length would crash with request typeof. In this statement, typeof is working off of dataTtypes.length which outputs number because .length would output the amount of elements/items in dataTypes as requested in numeric form. Therefore output is numbers.
```
Comment (chrisyhlee) [inline coding-concepts.js:61]: Great research and explanation here

**Review by chrisyhlee — APPROVED** [review]
Great job on your first assessment! You certainly have a strong understanding of the concepts here! I especially like that you are able to clearly explain your code as well as concepts presented in this assessment. This displays Advanced Beginner. One suggestion I have for you to work on is to make your pseudo code more concise. I left an example for you in the comments on challenge 1! You're doing really great so far, keep it up!

**Pull request merged**